### ISSUE TITLE
EMT: Nanosuit Goggles of Revealing does not give nightvision as description says.

### ISSUE THREAD

**Issue by Hachuurui** [GTNewHorizons/GT-New-Horizons-Modpack#8576]
### Your GTNH Discord Username

Hachy#6428

### Your Pack Version

2.1.1.0

### Your Server

TWPI MP  gtnh.twpi.pw

### Type of Server

Vanilla Forge

### Your Expectation

Putting on the goggles gives nightvision that cannot be turned off, like the tooltip says.

### The Reality

I get the ability to see thaumcraft stuff, but no nightvision.

### Your Proposal

See if its possible to get the nightvision added to it? This is from Electro magical tools mod. So I'm not sure if you have source access to adjust this.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Prometheus0000** [comment]
Did you try using the IC2 keybind(s)? Because I remember it working.

**Comment by Hachuurui** [comment]
Yes that was the first thing I tried. but let me try again.

**Comment by Hachuurui** [comment]
Its giving blindness :/

**Comment by Prometheus0000** [comment]
That means you're in the light, go somewhere dark.

**Comment by Hachuurui** [comment]
That is the most disorienting thing I've seen lol. ok thanks. I was using the wrong keybind. So I'm an idjiot

**Comment by bombcar** [comment]
I don’t get why tooltip says it can’t be turned off but it clearly can

**Comment by Prometheus0000** [comment]
Because the TT is wrong.

**Comment by GTNH-Afx237v7** [comment]
Have mentioned this here: https://github.com/GTNewHorizons/Hodgepodge/issues/20
